### PR TITLE
Use np.interp instead of interpolation.interp

### PR DESCRIPTION
**Pull request by kp992**
This PR replaces `interpolation.interp` to `np.interp` now that `numba` supports `np.interp` directly.  Also, removes `plt.rcParams["figure.figsize"] = (11, 5)  #set default figure size` redundant line.

### CODE REVIEW

**Comment by kp992** [comment]
@jstac @mmcky I am trying to replace `interp` function in this PR. There are a few more files which needs this change and will do that in new PR to keep this PR short and easy to review and verify. Thank you!

**Comment by jstac** [comment]
Many thanks @kp992 .

@mmcky or @HumphreyYang , could you please do a first round review of this?

The aim is that swapping to NumPy's interp should not change outputs or degrade performance.

**Comment by jstac** [comment]
Note that this is related to https://github.com/QuantEcon/lecture-python-advanced.myst/pull/156

**Review by mmcky — APPROVED** [review]
thanks @kp992 I confirm the `np.interp` has been updated nicely and the function arguments have been switched successfully.

**Comment by mmcky** [comment]
@jstac re: performance there are small degradations 

| lecture | current time | new time |
|--------|-------------|-----------|
| cake_eating_numerical | 30.4 | 33.8 |

**Comment by mmcky** [comment]
Remaining lectures to migrate

- [ ] lectures/jv.md:from interpolation import interp
- [ ] lectures/mccall_correlated.md:from interpolation import interp
- [ ] lectures/mccall_fitted_vfi.md:from interpolation import interp
- [ ] lectures/navy_captain.md:from interpolation import interp
- [ ] lectures/odu.md:from interpolation import mlinterp, interp
- [ ] lectures/optgrowth_fast.md:from interpolation import interp
- [ ] lectures/wald_friedman.md:from interpolation import interp

**Comment by mmcky** [comment]
@kp992 I know it violates our usual practice but let's try and fix them all in one PR -- so they are in one commit. Thanks.

File: lectures/odu.md [hunk 1]
```diff
@@ -58,9 +58,8 @@ Let’s start with some imports
 
 ```{code-cell} ipython
 import matplotlib.pyplot as plt
-plt.rcParams["figure.figsize"] = (11, 5)  #set default figure size
 from numba import njit, prange, vectorize
-from interpolation import mlinterp, interp
+from interpolation import mlinterp
```
Comment (mmcky) [inline lectures/odu.md:62]: @kp992 is there a corresponding function in `numpy` for `mlinterp`?

**Comment by mmcky** [comment]
thanks @kp992.

@jstac I think this is looking good -- and I have reviewed the function calls. @kp992 has done a nice job.

File: lectures/odu.md (hunk 1 above)
Comment (jstac) [reply lectures/odu.md:62]: No @mmcky, there's not.

There is in SciPy but it's not jit compilable.  Let's stick to interpolation for the multivariate case.

File: lectures/_static/lecture_specific/optgrowth/bellman_operator.py [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import numpy as np
-from interpolation import interp
```
Comment (jstac) [inline lectures/_static/lecture_specific/optgrowth/bellman_operator.py:2]: I didn't see `pip install interpolation` removed from this lecture... was that task forgotten?

**Comment by jstac** [comment]
Thanks @mmcky and great work @kp992 .

Please see minor comments above.

File: lectures/_static/lecture_specific/optgrowth/bellman_operator.py (hunk 2 above)
Comment (kp992) [reply lectures/_static/lecture_specific/optgrowth/bellman_operator.py:2]: Thanks @jstac. I think we can remove this file completely. This isn't use anywhere in any lectures. In `optgrowth`, it seems to be have been replace by `from scipy.interpolate import interp1d`.

```bash
% git grep bellman_operator
lectures/_static/lecture_specific/odu/odu.py:    def bellman_operator(self, v):
lectures/_static/lecture_specific/wald_friedman/wald_class.py:    def bellman_operator(self, J):
lectures/_static/lecture_specific/wald_friedman/wald_class.py:        J = qe.compute_fixed_point(self.bellman_operator, np.zeros(self.m),
lectures/_static/lecture_specific/wald_friedman/wf_first_pass.py:def bellman_operator(pgrid, c, f0, f1, L0, L1, J):
lectures/_static/lecture_specific/wald_friedman/wf_first_pass.py:bell_op = lambda vf: bellman_operator(pg, 0.5, f0, f1, 5.0, 5.0, vf)
```

File: lectures/odu.md (hunk 1 above)
Comment (kp992) [reply lectures/odu.md:62]: That's right, @jstac. I didn't find `mlinterp` in numpy too.

**Comment by mmcky** [comment]
thanks @jstac for searching the `py` includes. I had focused on the `md` 👍

File: lectures/odu.md (hunk 1 above)
Comment (mmcky) [reply lectures/odu.md:62]: @kp992 we will need `interpolation` package updated unless `mlinterp` doesn't use `generated_jit`. Would you have time to look into that?

**Comment by mmcky** [comment]
@kp992 are you happy for this to be merged?

**Review by mmcky — APPROVED** [review]

**Comment by kp992** [comment]
Thanks @mmcky and @jstac

**Pull request merged**